polyfill/README.md

```diff
@@ -0,0 +1,41 @@
+# Temporal Polyfill
+
+This is an implementation of a polyfill for the [TC39 Termporal Proposal](https://github.com/tc39/proposal-temporal).
+
+## Implementation Issues:
+
+### Edge-Case: Getting *epoch-milliseconds*
+
+Getting the correctly time-zoned *milliseconds since epoch* for a `CivilDateTime` when calling `withZone` is currently
+based on `Intl.DateTimeFormat` fumbling. However this fails for the edgecase where the instant to be extracted falls
+within one timezone offset of a DST-Change.
+
+The cost of implementing this correctly would be to bring in the TZ-Database (like momentjs has done); or alternatively using momentjs directly. However neither are worth the weight for the purpose of exploring the API.
+
+## Proposed Changes / Additions to the Proposal
+
+### `Instant.prototype.valueOf()` & `ZonedInstant.prototype.valueOf()`
+
+To facilitate comparisons between `Intant`s and/or `ZonedInstant`s it would be extremely useful to implement `valueOf()`.
+Considering the fact that these have nanosecond precision, the proposed `[BigInt](https://github.com/tc39/proposal-bigint)`
+seems to be the right choice of primitive.
```

Comment:
Yes! I was reluctant to depend on bigint when it was early stage, but its done at this point.